mods/ts/sequences/structures.yaml

```diff
@@ -1964,3 +1964,67 @@ gaplug2:
 
 gaplug3:
 	icon: rad3icon
+
+gafire:
+	Defaults:
+		Offset: -16, -28, 28
+		UseTilesetCode: true
+	idle:
+		ShadowStart: 3
+	damaged-idle:
+		Start: 1
+		ShadowStart: 4
+	dead:
+		Start: 2
+		ShadowStart: 5
+		Tick: 400
+	idle-lights: gtfire_c
+		Length: 6
+		ShadowStart: 5
+		Tick: 120
+	active: gtfire_b
+		Length: 16
+		ShadowStart: 16
+	opening: gtfire_a # TODO: unused
```

Comment:
What are these?

Reply:
Animations that don't play at the moment. `WithIdleOverlay` is a bit too limited.